docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/app_module/glossary/app-module--cgov-glossary-app.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display the glossary app_module.
+ *
+ * Available variables:
+ * - app_module_plugin_id: The ID of the app module implementation.
+ * - configuration: A list of the app module's configuration values.
+ * - content: The content of this app_module screen.
+ * - attributes: array of HTML attributes populated by modules, intended to
+ *   be added to the main container tag of this template.
+ *   - id: A valid HTML ID and guaranteed unique.
+ *
+ * @see template_preprocess_app_module()
+ *
+ * @ingroup themeable
+ */
+#}
+{{ attach_library('cgov_common/glossary-app') }}
+<noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
That is this file for vs the plugin templates?

Reply:
This was included because there is a similar template in cgov_cts. I assumed it was needed as a fallback in case the other templates didn't work. That isn't the case, so I've deleted it.